LTHPasscodeViewController/LTHPasscodeViewController.h

```diff
@@ -92,6 +92,10 @@
  */
 @property (nonatomic, weak) id<LTHPasscodeViewControllerDelegate> delegate;
 /**
+ @brief The count of digits for the simple passcode. By default passcode has 4 digits.
```

Comment:
I think _The number of digits_ is slightly more correct. Also, looking at other properties with default values, I'd say `Defaults to @c 4.` would be a more appropriate wording.